### DISCUSSION TITLE
Help when generating a production version

### DISCUSSION THREAD

**Discussion by hernan74** [CariusLars/ar_flutter_plugin#33] [Q&A] [answered]
Configure my project based on the guide to use the example of ExternalModelManagementWidget. I did the settings in my google and firebase console and it works for me. My problem arises when I release apk I get the following error "Error uploading anchor, state ERROR_NOT_AUTHORIZED"

**Comment by CariusLars** [comment, marked as answer]
That's probably the SHA-1 certificate fingerprint that you entered during the setup of your OAUTH2 project (as described in the guide). The key is different for debug and release and the guide only gives the command for the debug key. you can check [this stackoverflow answer](https://stackoverflow.com/questions/15727912/sha-1-fingerprint-of-keystore-certificate/35308827#35308827) on how to get the release key which should make it work in release mode
